lib/ansible/modules/web_infrastructure/jenkins_job.py

```diff
@@ -341,7 +341,7 @@ def test_dependencies(module):
 
 
 def job_config_to_string(xml_str):
-    return ET.tostring(ET.fromstring(xml_str))
+    return ET.tostring(ET.fromstring(xml_str)).decode('ascii')
```

Comment:
I'm not convinced by the fix: IMHO it assumes that there are no non-ASCII characters in the XML string. Without having gone into the depth of the XML library, there should be a way to get a str (instead of bytes) using the encoding from the XML header.

Reply:
@ericzolf please take a look at documentation of ElementTree.tostring: https://docs.python.org/2/library/xml.etree.elementtree.html#xml.etree.ElementTree.tostring -default encoding is 'us-ascii', so decode('ascii') should always work (ET.tostring encodes non-us characters as XML character entities).